### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by domenkozar** [thoughtpolice/hs-ed25519#20]
Hey @thoughtpolice, is there any chance we could get a release out? :)

**Comment by thoughtpolice** [comment]
Yes, the reason I didn't do it before is actually because my laptop died, and I still haven't (yet) recovered my GPG private key on my new laptop. Sorry about that!

Unfortunately this past week and this week I'll likely be entirely locked up with things to do (family/friend-wise) so it may have to wait until next week. Or maybe I can do it late tonight if I shave off the time...

**Comment by domenkozar** [comment]
Did you manage to get those GPG keys? :)

**Comment by domenkozar** [comment]
🙏

**Comment by domenkozar** [comment]
🍰

**Comment by parsonsmatt** [comment]
:pizza:

**Comment by domenkozar** [comment]
:shipit:

**Comment by thoughtpolice** [comment]
@domenkozar Hey Domen, I apologize about this. My laptop recently failed (despite being a year old...) and I'm looking to clean this up, so I have to recover my key *again*. Then again, I don't think my GPG key had an expiry, either, so maybe I should spin a new one... I've also got just... a bunch of things I'm tidying up. 

Anyway, this is pretty clearly an instance of "Maintainer absence" more or less, even if the package works fine. I *will* return to this work, but it's so far outside my critical path at the moment that it's just not been a priority. However, I understand it is problematic to keep things around for users like you.

Thus, I'd like to give you co-maintainer role for both the GitHub repository, and the Hackage package. I believe there is nothing wrong with the current state of the repository. As an extra layer of security, I'd also like to make @hvr a third-level maintainer, in case both of us mysteriously vanish off Earth (Cabal is _the_ most important user of this package, and Herbert is well in-tune with it -- so I feel this is an OK idea that won't massively explode his responsibilities, and is more of an insurance policy. :)

If you're OK with this, please just respond here and I'll add you. All I'd ask is that you might CC me before a release or major change (either here or by email or twitter or whatever). Please fill out the changelog, try to be tidy, clean up aggressively and don't be afraid to deprecate, etc etc (my major plan eventually was to keep everything deprecated available until a 0.1 release, which will remove all the old deprecated APIs, JFYI.). I trust you on this, just ask if you're unsure about anything.

**Comment by domenkozar** [comment]
Thanks @thoughtpolice, I understand, it's open sourse at the end of the day.

@parsonsmatt do you want to take over given that it's mostly in IOHK interest to clean this up?

**Comment by thoughtpolice** [comment]
Here is the current delta between `v0.0.5.0` and `master`: https://github.com/thoughtpolice/hs-ed25519/compare/v0.0.5.0...master

**Comment by parsonsmatt** [comment]
Thanks for all of your work @thoughtpolice -- it's much appreciated :)

I'll ping the folks at IOHK and see if any of them would be interested in co-maintainership. I would be happy to do so, but this is a bit outside of my personal wheelhouse, and I suspect there's someone more suitable on the team.
